### PR TITLE
Add 115 cloud v4.1.0

### PR DESCRIPTION
**Pull request by githubutilities**

### CODE REVIEW

File: Casks/115cloud.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+  url "http://pc.115.com/download/mac/115com_mac_#{version}.dmg"
+  name '115网盘'
+  name '115 cloud'
+  homepage 'http://pc.115.com/mac.html'
+  license :gratis
+
+  app '115网盘.app', :target => '115 cloud.app'
+  uninstall :quit => '115comonlinestorage-15--'
+  zap :quit => '115comonlinestorage-15--',
```
Comment (vitorgalvao) [inline Casks/115cloud.rb]: `zap` always calls `uninstall` first, so no need to repeat.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in the below referenced pull request. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by githubutilities** [comment]
Can I ask you how you manage complicated git workflow(fixing issues, merging PR, discussion) while keeping commit credited to original author? Do you use pure `git` command?

**Comment by vitorgalvao** [comment]
> Do you use pure `git` command?

Yes, it is all done with standard `git` commands. However, since it’s something I do so often, I’ve made [a script](https://github.com/vitorgalvao/tiny-scripts/blob/master/prfix) to speed up the process.

It consists solely of pulling your changes, making mine, and then rebasing everything into your original commit.

**Comment by githubutilities** [comment]
Thanks.

**Pull request closed without merging**